make resizefs faster (#956)
topolvm/topolvm: docs/topolvm-controller.md

### The Controller for Nodes

The controller is to cleanup all PVCs and LogicalVolumes associating to the deleting Node.

> It adds `topolvm.io/node` finalizer to run the cleanup task.

Reviewer: ```suggestion
It adds the `topolvm.io/node` finalizer to run the cleanup task.
```

This node finalize procedure may be skipped with the `--skip-node-finalize` flag. 
When this is true, the PVCs and the LogicalVolume CRs from a deleted node must be
deleted manually by a cluster administrator.

### The Controller for PersistentVolumeClams

> The controller is to accomplish two tasks.

Reviewer: ```suggestion
The controller accomplishes two tasks.
```

> The first is to delete Pods using deleting PVCs.

Reviewer: ```suggestion
1. Delete Pods using PVCs under deletion.
```

When a PVC for TopoLVM is being deleted, the controller deletes pods referencing
the PVC, if any. This is repeated until other finalizers to be completed.
Once it becomes the last finalizer, it removes the finalizer to immediately delete the PVC.

> The second is to speed up resizing a filesystem by kubelet.

Reviewer: ```suggestion
2. Speed up resizing a PVC filesystem by nudging the kubelet.
```

kubelet watches Pods rather than PVCs periodically to resize the filesystem,
therefore the filesystem resizing may be delayed.
To avoid this, the controller will notify kubelet by setting

> the `topolvm.io/resizefs-requested-at` annotation with the current time to the Pod.

Reviewer: ```suggestion
the `topolvm.io/last-resizefs-requested-at` annotation with the current time to the Pod.
```
